plugins/github-actions/package.json

```diff
@@ -22,6 +22,7 @@
   },
   "dependencies": {
     "@backstage/core": "^0.1.1-alpha.13",
+    "@backstage/core-api": "^0.1.1-alpha.13",
```

Comment:
Don't use this, import from core instead. All exports from core-api are re-exported by core.

Reply:
Ah. I see. We still use this import on some other places.